### ISSUE TITLE
Bad CPU type in executable

### ISSUE THREAD

**Issue by BirjuVachhani** [pingbird/puro#101]
Getting this error trying to install Puro on a fresh Mac M1 Pro.

Command:
```bash
curl -o- https://puro.dev/install.sh | PURO_VERSION="1.4.6" bash
```

Error:
```
bash: line 69: /Users/user/.puro/bin/puro.new: Bad CPU type in executable
```

**Comment by pingbird** [comment]
Does it work after installing Rosetta? `softwareupdate --install-rosetta`

The install script is supposed to automatically detect that, I wonder if a recent MacOS update broke it.

**Comment by pingbird** [comment]
One more thing, what does `uname -m` output?

**Comment by BirjuVachhani** [comment]
Yeah, I saw the script and figured out that it was failing because of no rosetta. I installed it manually and it worked.
